Automattic/jetpack
# Donations block: prevent crash on currency change

Fixes #30299 (donation block currency changes crashed the block)

## Proposed changes:

* Adds `removeEventListener` to Amount blur.
* Updates the default amounts when the change occurs.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On sandbox run `bin/jetpack-downloader test jetpack earn/fix/change-donation-block-currency`
* Sandbox the site you want to check on and load the donations block.  Change the currency.  The block should not crash.
* There exist some other issues with the amount component, but these will be addressed in other patches.
